src/app/(frontend)/api/stripe/webhooks/route.ts

```
@@ -0,0 +1,104 @@
+      switch (event.type) {
+        case "checkout.session.completed":
+          data = event.data.object;
+
+          if (!data.metadata?.userId) {
+            throw new Error("User ID not found");
+          }
+
+          const user = await payload.findByID({
+            collection: "users",
+            id: data.metadata.userId,
+          });
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)